src/en/authors-charm-metadata.md

```diff
@@ -125,23 +125,33 @@ resources:
 
 ## Payloads
 
-You can use the `payloads` section of metadata.yaml to help the user of a charm
-better understand the purpose of payloads such as LXC, KVM and docker. This is
-especially useful in large and complex deployments.
+Payloads provide a means for the charm author to get information out of the
+services deployed by a charm. This is especially useful in large and complex
+deployments. For instance, the author may want to check the status of some
+element of the services such as a Docker container.
 
-Payloads are defined by creating a class for the payload, such as `monitoring`
-or `kvm- guest`, and then assigning a type:
+Payloads are defined in the `payloads` section of `metadata.yaml` by assigning
+a class and type. A class defines the name of the payload and the type
+describes the nature of the payload. Both are author-defined and are not
+validated by Juju.
+
+The most common types of payload are based on Docker, KVM, and LXD.
+
+As an example, below the following class/type pairs are defined:
```

Comment:
for some reason I feel strongly there should be an additional ',' after below

Reply:
OK